src/components/ChatLog.js

```diff
@@ -0,0 +1,33 @@
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+
+    const chatComponents = props.entries.map(entry => {
```

Comment:
So you see how setting all of those props took up a lot of lines of code? We could make that more concise with this here: 
```javascript
 < ChatEntry toggleLike={props.toggleLike} key={entry.id} entry={entry} />
```
Then in each `chatEntry` we could access any prop we needed with `props.entry.myProp`